src/components/atoms/Icon/Component.tsx

```diff
@@ -49,16 +37,17 @@ const Icon: FunctionComponent<IconProps> = forwardRef<any, IconProps>(
     const iconSpecificStyle = getIconStyle(type, {});
 
     return (
-      <IconComponent
-        ref={ref}
-        testID="iconIcon"
-        style={StyleSheet.flatten([styles.icon, style && style])}
-        size={size}
-        name={name}
-        color={color}
-        {...iconSpecificStyle}
-        {...props}
-      />
+      <View ref={ref} style={style}>
```

Comment:
Yes it has but seem not stable, sometimes it doesn't work and make project breaking. That's why I wrapped it

Reply:
ok ok. If it not stable good to go. But keep in mind, the library should have fewer components because passing props with many style objects isn't a good idea, you have more work in Jitera tool